src/transformation/intrinsic_transformation.c

```diff
@@ -248,8 +255,10 @@ static k4a_result_t transformation_unproject_internal(const k4a_calibration_came
         return K4A_RESULT_FAILED;
     }
 
-    if (camera_calibration->intrinsics.type == K4A_CALIBRATION_LENS_DISTORTION_MODEL_RATIONAL_6KT)
+    if (camera_calibration->intrinsics.type == K4A_CALIBRATION_LENS_DISTORTION_MODEL_RATIONAL_6KT &&
+        g_deprecated_6kt_message_fired == false)
     {
+        g_deprecated_6kt_message_fired = true;
         LOG_CRITICAL("Rational 6KT is deprecated (only supported early internal devices). Please replace your Azure "
```

Comment:
Do we want to log the information as Critical level? Maybe warning will be enough?

Reply:
The issue is still critical for customers, so it should stay critical. If the calibration turns up then a production device is needed.

Reply:
I see. It makes sense :)